ntbs-integration-tests/Helpers/TestDocumentExtensions.cs

```diff
@@ -13,13 +15,35 @@ public static string GetError(this IHtmlDocument document, string input)
             return errorSpan.TextContent;
         }
 
+        public static string GetErrorFromSummary(this IHtmlDocument document, string errorSummaryName, string errorSpanName)
+        {
+            var errorLink = (IHtmlAnchorElement) document?.QuerySelector($"a[id='error-summary-{errorSummaryName}']");
+            if (errorLink == null) return null;
+            var splitHrefLink = errorLink.Href.Split("#");
+            var errorParentId = splitHrefLink[splitHrefLink.Length - 1];
+            var errorParent = document.QuerySelector($"#{errorParentId}");
+            var errorSpan = errorParent.QuerySelector($"span[id='{errorSpanName}-error']");
+            if (errorSpan == null) return null;
+            if (errorSpan.ClassList.Contains("hidden")) return null;

```

Comment:
Stylistic choice, but I would probably wrap these into the same `if`
e.g. 
`if (errorSpan == null || errorSpan) 
{
   return null
}`